lib/looker-sdk/response/raise_error.rb

```diff
@@ -30,9 +30,7 @@ module LookerSDK
   module Response
 
     # HTTP status codes returned by the API
-    class RaiseError < Faraday::Response::Middleware
-
-      private
```

Comment:
new middleware interface: needs to sublcass `Faraday::Middleware` and publicly expose `on_complete`